addons/website/static/src/snippets/s_website_form/form.edit.js

```diff
@@ -1,19 +1,18 @@
 import { registry } from "@web/core/registry";
+import { Interaction } from "@website/core/interaction";
+import { patch } from "@web/core/utils/patch";
 import {
     formatDate,
     formatDateTime,
 } from "@web/core/l10n/dates";
 const { DateTime } = luxon;
 import { Form } from "./form";
 
-const FormEdit = I => class extends I {
-    setup() {
-        super.setup();
-        this.editTranslations = this.services.website_edit.isEditingTranslations();
-    }
+export class FormEdit extends Interaction {
+    static selector = ".s_website_form form, form.s_website_form"; // !compatibility
 
-    prepareDateFields() {
-        // We do not initialize the datetime picker in edit mode but want the dates to be formated
+    start() {
+        // We do not initialize the datetime picker in edit mode but want the dates to be formatted.
         this.el.querySelectorAll(".s_website_form_input.datetimepicker-input").forEach(el => {
```

Comment:
not sure, but if you want, an alternative is to have a separate interaction for each datetimepicker-input instead